shell/platform/embedder/tests/embedder_unittests.cc

```diff
@@ -2766,6 +2766,45 @@ TEST_F(EmbedderTest, CanSendPointerWithViewId) {
   message_latch.Wait();
 }
 
+TEST_F(EmbedderTest, WindowMetricsEventDefaultsToImplicitView) {
```

Comment:
This isn't testing `embedder.cc`'s `SAFE_ACCESS`, this is testing C++'s zero initialization of structs. I don't think this test is useful in its current state.

If you want to test `SAFE_ACCESS`, I would create a new struct that has the same members as `FlutterWindowMetricsEvent` except for the `view_id`, and then doing a `reinterpret_cast<FlutterWindowMetricsEvent*>(&event)`.

Reply:
That's a good point. I've changed it to use a struct_size right before view_id (I've tested by changing embedder.h's default value that it works). Do you think the test makes sense or do you think it's still kind of unnecessary? (I'm not strong on adding this; I thought in our discussion you said to test the behavior for the implicit view.)

Reply:
Oh nice, changing the `struct_size` works too! Marking as resolved